### ISSUE TITLE
v1.18.1 fails on vercel

### ISSUE THREAD

**Issue by mashaal** [vercel-community/deno#99]
Upgrading to deno `1.18.1` fails when deploying to vercel.

Log output:

```
Downloading `deno` v1.18.1…
--
16:36:33.936 | Installed `deno`:
16:36:33.936 | deno: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by deno)
16:36:33.940 | Error: Build script failed with exit code 1
```

**Comment by TooTallNate** [comment]
Ya, I can reproduce. Looks like this is already being tracked by deno [here](https://github.com/denoland/deno/issues/13516).

**Comment by kidonng** [comment]
Deno just released [1.19.3](https://github.com/denoland/deno/releases/tag/v1.19.3) which fixed the upstream issue. Tested on Vercel and the build succeeds.

**Comment by TooTallNate** [comment]
Update default Deno version to 1.20.1 in #105, released in `vercel-deno@1.1.2`, so I will close this issue.
